server/schema.sql

```diff
@@ -1 +1 @@
-create table if not exists user( id integer primary key autoincrement, username text not null, password text not null, key blob default None );
\ No newline at end of file
+create table if not exists user( id integer primary key autoincrement, username text not null, password text not null, key blob default None,cookie text default None );
```

Comment:
A user could have multiple session_tokens. This table structure does not take that into account.

Reply:
why we can store many cookies with same username and password. so if person login with different cookie it will work

Reply:
Let's say if a user joins from a laptop and a computer. If both of them use same `session_token`, then there is no way to logout one without the other (from server side).

So generally we create a new `token` if user logs in from a new device (or new browser etc.)

Here we probably want a new table like

```md
id | user_id | token
---|---------|----------
1  | 3       | "token-1"
2  | 3       | "token-2"
3  | 5       | "token-3"
3  | 2       | "token-4"
```